### PR TITLE
Feature/responsiveness

### PR DESCRIPTION
**Pull request by anetSzajo**

### CODE REVIEW

File: src/Components/SharedComponents/PlantPhoto/PlantPhoto.js [hunk 1]
```diff
@@ -1,12 +1,118 @@
...
+                console.log(err)
+            })
+    }
+
+    uploadPlantImage = (photo) => {
+        const data = new FormData();
+        data.append('image', photo);
+
+        (async () => {
+            axios.interceptors.response.use(
```
Comment (kSzajo) [inline src/Components/SharedComponents/PlantPhoto/PlantPhoto.js:52]: interceptors should be configured near root level. As we agreed move this logic inside catch of post

File: src/Components/SharedComponents/PlantPhoto/PlantPhoto.js [hunk 2]
```diff
@@ -1,12 +1,118 @@
...
+        axios.delete(this.state.plant.imageUrl)
+            .then(res => {
+                console.log("Photo Deleted Successfully.")
+            })
+            .then(res => {
+                this.setState({
+                    imgSrc: '/icons/defaultPlantPhoto.png'
+                })
+            })
+            .catch(err => console.log(err))
```
Comment (kSzajo) [inline src/Components/SharedComponents/PlantPhoto/PlantPhoto.js:83]: please add snackbar with alert if delete failed

File: src/Components/SharedComponents/PlantPhoto/PlantPhoto.js (hunk 1 above)
Comment (anetSzajo) [reply src/Components/SharedComponents/PlantPhoto/PlantPhoto.js:52]: Fixed

File: src/Components/SharedComponents/PlantPhoto/PlantPhoto.js (hunk 2 above)
Comment (anetSzajo) [reply src/Components/SharedComponents/PlantPhoto/PlantPhoto.js]: Fixed

File: src/Components/PlantPage/PlantFullDescription/plantFullDescription.scss [hunk 3]
```diff
@@ -6,26 +6,15 @@
   margin: 1.5rem auto;
   padding: 1.5rem;
   position: relative;
-  width: 70%;
+  //width: 70%;
```
Comment (kSzajo) [inline src/Components/PlantPage/PlantFullDescription/plantFullDescription.scss:9]: remove commented code

File: src/Components/SharedComponents/AppLogo/styles.scss [hunk 4]
```diff
@@ -1,16 +1,20 @@
...
+  align-items: center;
   font-family: $font-family;
   font-size: 13px;
   color: white;
   background-color: $second-color;
-  position: absolute;
-  top: 19px;
-  left: 35%;
+  //position: absolute;
+  //top: 5%;
```
Comment (kSzajo) [inline src/Components/SharedComponents/AppLogo/styles.scss:13]: remove commented code

**Review by kSzajo — APPROVED** [review]

File: src/Components/PlantPage/PlantFullDescription/plantFullDescription.scss (hunk 3 above)
Comment (anetSzajo) [reply src/Components/PlantPage/PlantFullDescription/plantFullDescription.scss]: Removed

File: src/Components/SharedComponents/AppLogo/styles.scss (hunk 4 above)
Comment (anetSzajo) [reply src/Components/SharedComponents/AppLogo/styles.scss]: Removed.

**Pull request merged**